src/App.js

```diff
@@ -159,6 +160,7 @@ const routeConfig = [
   { path: "/AnimalsDash/AnimalsMeowFacts", component: AnimalsMeowFacts },
   { path: "/AnimalsDash/AnimalsRandomDog", component: AnimalsRandomDog },
   { path: "/AnimalsDash/AnimalsShibeOnline", component: AnimalsShibeOnline },
+  { path:"/AnimalsDash/AnimalCatFacts", component:Catfacts},

```

Comment:
Updat to AnimalsCatFacts

Reply:
I have made the necessary changes could you please look into it